examples/async/index.js

```diff
@@ -1,15 +1,43 @@
 import 'babel-polyfill'
 import React from 'react'
-import { render } from 'react-dom'
+import ReactDOM from 'react-dom'
 import { Provider } from 'react-redux'
-import App from './containers/App'
 import configureStore from './store/configureStore'
 
 const store = configureStore()
+const rootEl = document.getElementById('root')
 
-render(
-  <Provider store={store}>
-    <App />
-  </Provider>,
-  document.getElementById('root')
-)
+let render = () => {
+  const App = require('./containers/App').default
```

Comment:
The reason it’s an inline require is because we want to import *the latest version* after hot update.
Babel transpiles `import` to `require()` so we have no opportunity to `require()` again.

In Webpack 2, if we disable Babel modules transform, we can use a normal `import` because Webpack will keep it a live binding and update the imported value.